games/f.yaml

```
@@ -342,6 +342,27 @@
   type: remake
   url: http://www.freecol.org/
 
+- name: FreeCS
+  images:
+  - https://a.fsdn.com/con/app/proj/freecs-1-5/screenshots/screen1.png/max/max/1
+  - https://a.fsdn.com/con/app/proj/freecs-1-5/screenshots/screen2.png/max/max/1
+  - https://a.fsdn.com/con/app/proj/freecs-1-5/screenshots/screen3.png/max/max/1
+  - https://a.fsdn.com/con/app/proj/freecs-1-5/screenshots/screen4.png/max/max/1
+  - https://a.fsdn.com/con/app/proj/freecs-1-5/screenshots/portable.jpg/max/max/1
+  lang: QuakeC
+  license:
+  - GPL2
+  development: sporadic
+  originals:
+  - Counter-Strike
+  repo: https://sourceforge.net/projects/freecs-1-5/
+  type: remake
+  content: free
+  multiplayer:
+  - Online
+  - LAN
+updated: 2019-06-11
```

```suggestion
  updated: 2019-06-11
```
oops, got the indenting wrong